index.html

```diff
@@ -17,75 +17,89 @@
   <body>
     <main>
       <header>
+        <div class="nav-title">
+          <h3><a href="index.html">Ismail Changezi</a></h3>
+        </div>
         <nav>
-          <h3>Ismail Changezi</h3>
+          <ul class="nav-items">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li>
+              <a href="#"><img src="img/mail.png" alt="" /></a>
+            </li>
+          </ul>
           <i class="fa-solid fa-bars"></i>
         </nav>
-        <section class="header-section">
+      </header>
+      <section class="header-section">
```

Comment:
- [ ] please ensure your work is responsive. An extra whitespace appears while switching to the desktop version
![Screenshot (406)_LI](https://user-images.githubusercontent.com/65298048/188021629-1d858fd7-f9a0-4dfb-a3d2-07b84882d50c.jpg)

![Screenshot (407)_LI](https://user-images.githubusercontent.com/65298048/188021825-59c3b315-0d9a-4560-9cb1-b9205fc885aa.jpg)

Reply:
I dont know why you are facing this issue it works fine on my screen
![(1) PRP Treatment for Hair Loss WORTH IT!_ Experiences, Reults, Price - YouTube - Google Chrome 9_2_2022 3_46_50 PM](https://user-images.githubusercontent.com/96125730/188123786-b3294fea-7731-4706-9d3d-99384b886ae3.png)

Reply:
And as far as the alignment of cards were concerned I have worked on it and now its working great